lib/node_modules/@stdlib/strided/base/map-by2/test/test.main.js

```diff
@@ -72,7 +72,8 @@ tape( 'the function applies a binary function to indexed strided array elements
 	mapBy2( x.length, x, 1, y, 1, z, 1, add, accessor );
 	t.deepEqual( z, expected, 'deep equal' );
 
-	x = new Array( 5 ); // sparse array
+	x = [ ]; // sparse array
+	x.length = 5;
```

Comment:
```suggestion
	// Create a sparse array:
	x = new Array( 5 ); // eslint-disable-line stdlib/no-new-array
```

In this case, the use of `new Array` is intentional, so we can just disable the lint rule.